mappings/net/minecraft/network/packet/client/CPacketWorldEvent.mapping

```diff
@@ -1,21 +1,21 @@
-CLASS none/gw net/minecraft/network/packet/client/CPacketEffect
-	FIELD a effectId I
+CLASS none/gw net/minecraft/network/packet/client/CPacketWorldEvent
+	FIELD a eventId I
```

Comment:
It's still a bit ambiguous if `id` is referring to the event ID or something else, like world ID. I was more thinking of renaming `data` to `eventData`, actually.